lib/mixlib/install/options.rb

```diff
@@ -0,0 +1,36 @@
+module Mixlib
+  class Install
+    module Options
```

Comment:
Thinking about this again, I think we should make this a class with some readers to get the option values. It should validate the parameters in its constructor. 

Previously I was thinking we can mixin this into the classes we want to support the options. But thinking about the `self.info` method above, this will not be really convenient since class methods will not have access to options via the mixed helpers.